services/src/major_recommendation_logic.py

```diff
@@ -0,0 +1,54 @@
+import pandas as pd
+from sentence_transformers import SentenceTransformer
+from sklearn.metrics.pairwise import cosine_similarity
+from .data_loader import DataLoader
+from .train_model import TrainModel
+
+
+class MajorRecommendationLogic:
```

Comment:
Can we add docstring to the class and its methods?

Reply:
Added docstrings wherever required.